chaincfg/params.go

```diff
@@ -227,6 +227,11 @@ type Params struct {
 	// BIP44 coin type used in the hierarchical deterministic path for
 	// address generation.
 	HDCoinType uint32
+
+	// SLP indexer parameters
+	SlpIndexStartHeight uint32
+	SlpIndexStartHash   []byte
```

Comment:
Let's use a `chainhash.Hash` for this instead of a byte slice. 

It can be initialized using `newHashFromStr()`

Reply:
done.